### ISSUE TITLE
iso failed to install for ublue silverblue main

### ISSUE THREAD

**Issue by dnkmmr69420** [ublue-os/main#106]
![Screenshot from 2023-04-04 09-04-20](https://user-images.githubusercontent.com/104856296/229818630-cb7e4fb5-42a4-4890-8e75-f92d0ea8dd5b.png) this says it all

**Comment by castrojo** [comment]
I've seen this a few times, can you attach the anaconda log? https://docs.fedoraproject.org/en-US/quick-docs/anaconda/anaconda_logging/

**Comment by castrojo** [comment]
Found the issue (or at least a similar error), the machine needs to be on the network and if it's not it'll put this message up, I'll add this to the docs.

**Comment by tunix** [comment]
I got the same error. I'm trying the ISO via Boxes and the VM seems to be on the network via NAT?

**Comment by joshua-stone** [comment]
This is a common issue with the github container registry and isn't necessarily a bug that's intrinsic what we're doing with the netinstaller. I get it frequently when testing in a VM, and this is unlikely to change until anaconda adds the ability to retry image pulls and/or ublue-os moves images to a more reliable registry host.

**Comment by deiviuds** [comment]
I've just experienced the same error installing on hardware, trying kinoite-nvidia:38, will be trying another one now.
Using ISO v1.2.1 and connected to a good wifi signal

**Comment by deiviuds** [comment]
> I've just experienced the same error installing on hardware, trying kinoite-nvidia:38, will be trying another one now. Using ISO v1.2.1 and connected to a good wifi signal PS: I was trying to break the rules by installing on a partition, to later dual boot, but I don't think this signature is related to that

Apparently I was wrong or I had some issues with my network.
1. I tried again, and exit the network connection unconnected, to finish other things and the password kept popping asking to fill again, haven't happened before doing the same thing. Just exited and in the next time:
2. I waited to make sure the network was connected, then I formatted everything, and the installation was successful.

Not sure what happened, but if this is the expected error, it could be on the documentation, saying "if this then that" with common errors, for those stubborn users or regular one's haha
